locales/zh_CN/LC_MESSAGES/meta.po

```diff
@@ -90,25 +108,28 @@ msgstr ""
 
 #: ../../meta.rst:47
 msgid "For sparse tensors, the full domain shape will be returned."
-msgstr ""
+msgstr "对稀疏张量而言，此处会返回其完整域的形状（full domain shape）。"
 
 #: ../../meta.rst:50
 msgid "Compile-time evaluations"
-msgstr ""
+msgstr "编译时求值（evaluation）"
 
 #: ../../meta.rst:51
 msgid ""
 "Using compile-time evaluation will allow certain computation to happen "
-"when kernels are instantiated. Such computation has no overhead at "
-"runtime."
+"when kernels are instantiated. Such computation has no overhead at runtime."
 msgstr ""
+"使用编译时求值可以使得某些计算在内核被实例化时发生。这些计算不会有运行时开"
+"支。"
 
 #: ../../meta.rst:54
 msgid ""
-"Use ``ti.static`` for compile-time branching (for those who come from "
-"C++17, this is `if constexpr "
-"<https://en.cppreference.com/w/cpp/language/if>`_.)"
+"Use ``ti.static`` for compile-time branching (for those who come from C+"
+"+17, this is `if constexpr <https://en.cppreference.com/w/cpp/language/"
+"if>`_.)"
 msgstr ""
+"在编写编译时的分支条件时，请使用 ``ti.static`` （对使用C++17的人来说，这就"
```

Comment:
```suggestion
"用 ``ti.static`` 来进行编译时的分支展开（对C++17的用户来说，这相当于"
```

Reply:
Like this change. I was really struggling when translating this sentence.